Content.Server/Ame/AmeNodeGroup.cs

```diff
@@ -177,6 +173,18 @@ public float InjectFuel(int fuel, out bool overloading)
         return powerOutput;
     }
 
+    /// <summary>
+    /// Calculates the amount of power the AME can produce with the given settings
+    /// </summary>
+    public float CalculatePower(int fuel, int cores)
+    {
+        // Note the float conversions. The maths will completely fail if not done using floats.
+        // Oh, and don't ever stuff the result of this in an int. Seriously.
+        var floatFuel = (float) fuel;
+        var floatCores = (float) cores;
+        return 20000f * floatFuel * floatFuel / floatCores;
```

Comment:
I know you're just carrying over code that somebody else wrote, but this is not necessary. See: https://color-of-code.de/programming/c-sharp/numeric-promotions

Reply:
You are right. I don't know why whoever wrote this was so adamant about this stuff... I replaced the comments with some that explain some effects of this formula on power scaling